### ISSUE TITLE
Can't type - (minus) character

### ISSUE THREAD

**Issue by codemonkeykong** [Ylianst/MeshCommander#108]
Using MeshCommander version 0.9.5 (NPM in Firefox under Ubunntu)

In Remote Desktop window after connecting to Debian server can't type - (minus) character.

**Comment by kaysond** [comment]
Related: https://github.com/Ylianst/MeshCentral/issues/7127

**Comment by kaysond** [comment]
Looks like this was fixed in #113

**Comment by si458** [comment]
yep this should be fixed with the new install of 0.9.7 msi but not the npm repo version yet
https://meshcentral.com/meshcommander/MeshCommander-0.9.7.msi
